scripts/multiple_csv_upload.py

```diff
@@ -1,26 +1,22 @@
 import os
 import pathlib
 
-import environ_setup
 import requests
 
-environ_setup.setup()
+from dotenv import load_dotenv
 
-MODEX_URL = "https://modex.rl-institut.de/upload/"
+load_dotenv()
 
-path = pathlib.Path(
-    "/home/local/RL-INSTITUT/felix.maurer/rli/Felix.Maurer/SEDOS/Python/data_adapter_oemof/tests/"
-    "_files/tabular_datapackage_mininmal_example_collection/csvs"
-)
+path = pathlib.Path("Path to csvs")
 
 CSVS = list(path.iterdir())
 
-TOKEN = os.environ["TOKEN"]
-
+TOKEN = os.getenv("TOKEN")
```

Comment:
This is good - once you refactored script into function, you could set token as parameter to function and if it's not set, try to read it from env (this is the most user friendly option IMO)